config/settings/common.py

```diff
@@ -153,7 +153,10 @@
         # See: https://docs.djangoproject.com/en/dev/ref/settings/#std:setting-TEMPLATES-BACKEND
         'BACKEND': 'django.template.backends.django.DjangoTemplates',
         # See: https://docs.djangoproject.com/en/dev/ref/settings/#template-dirs
-        'DIRS': [],
+        'DIRS': [
+            str(APPS_DIR.path('main/templates')),
+            str(APPS_DIR.path("main/templates_inject"))
```

Comment:
Skoro potrzebujesz to jest to ok.

Reply:
Powtórzenie https://github.com/watchdogpolska/bliski_publikator/blob/9cecdbeca8b1590b886c3e756b1335d9d61ef6b2/config/settings/common.py#L187